pkg/webhook/admission/pod/mutator.go

```diff
@@ -29,10 +28,13 @@ import (
 	"github.com/kubeflow/kfserving/pkg/controller/inferenceservice/resources/credentials"
 	"github.com/kubeflow/kfserving/pkg/webhook/third_party"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/types"
 )
 
+var log = logf.Log.WithName("kfserving-mutator")
```

Comment:
We're not using log, klog, and logf. Can we leave the log out until we sort out which one we're going to use everywhere?

Reply:
Sure!